thing-url-adapter.js

```diff
@@ -117,6 +116,7 @@ class ThingURLDevice extends Device {
     this.notifiedEvents = new Set();
     this.scheduledUpdate = null;
     this.closing = false;
+    this.adapter = adapter;
```

Comment:
This is unnecessary, as it's already done in the base class.

Reply:
I figured it was there somewhere, I did not know how to access it. 

How would I access it?

Reply:
The same way. `this.adapter` should already be present.

Reply:
https://github.com/WebThingsIO/gateway-addon-node/blob/master/lib/device.js#L23

Reply:
removed